androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/common/HorizontalPagerIndicator.kt

```diff
@@ -0,0 +1,101 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.common
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.pager.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.*
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+import kotlin.math.absoluteValue
+import kotlin.math.sign
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun HorizontalPagerIndicator(
+    pagerState: PagerState,
+    pageCount: Int,
+    modifier: Modifier = Modifier,
+    pageIndexMapping: (Int) -> Int = { it },
+    activeColor: Color = LocalContentColor.current.copy(alpha = LocalContentAlpha.current),
+    inactiveColor: Color = activeColor.copy(ContentAlpha.disabled),
+    indicatorWidth: Dp = 8.dp,
+    indicatorHeight: Dp = indicatorWidth,
+    spacing: Dp = indicatorWidth,
+    indicatorShape: Shape = CircleShape,
+) {
```

Comment:
FYI, we have page indicator from `Accompanist`. Not sure if it is you want. If it is, you can remove this class and use the one from it.

Reply:
@kaungkhantsoe I'm using the pager from compose foundation. Pageindicator from Accmpanist is not compatible with that. So I basically copy-pasted it from accompanist.